cli/harness/src/test/java/io/helidon/build/cli/harness/CommandParserTest.java

```diff
@@ -308,6 +310,7 @@ public void testArgsFileOptionWithIncorrectFile() {
     }
 
     @Test
+    @DisabledOnOs(value = OS.WINDOWS, disabledReason="Fails to parse 'test-props-file.properties' absolute path")
```

Comment:
It's not a limitation, it's a bug right ?

It is about using `TestFiles.testResourcePath` for `argsFilePath` below ?

Reply:
Not sure it is, I have to look deeper into the issue (I am not familiar with the code involved yet).

Here is the stack trace:

```
Error:  io.helidon.build.cli.harness.CommandParserTest.testPropsFileOptionWithExistingFile  Time elapsed: 0.016 s  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /D:/a/helidon-build-tools/helidon-build-tools/cli/harness/target/test-classes/io/helidon/build/cli/harness/test-props-file.properties
	at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
	at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:232)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at io.helidon.build.cli.harness.CommandParser.parseCommand(CommandParser.java:247)
	at io.helidon.build.cli.harness.CommandParser.parseCommand(CommandParser.java:204)
	at io.helidon.build.cli.harness.CommandParserTest.testPropsFileOptionWithExistingFile(CommandParserTest.java:317)
```

Reply:
The test is building a `file://` URI for a test resource and passes it as a cli argument. If you update the test to use `TestFiles.testResourcePath` it should work.